simplified-main/src/main/java/org/nypl/simplified/main/MainServices.kt

```diff
@@ -389,7 +389,7 @@ internal object MainServices {
     return try {
       this.createBundledCredentials(context.assets)
     } catch (e: FileNotFoundException) {
-      this.logger.debug("could not initialize bundled credentials: ", e)
+      this.logger.debug("could not initialize bundled credentials; none found")
```

Comment:
At some point in the future, I'm going to try to introduce something where you have to state that you meant to include bundled credentials. If you state that you meant to include them and you don't, the app should refuse to start up. If you don't state that you meant to include them, and you actually do include the file, the app should also refuse to start up.

Right now we just print an exception regardless of your actual intention (and now we don't even do that!).

Reply:
Yes, that sounds like a nice improvement.